components/Feature/VulnerabilitiesGrid/index.js

```diff
@@ -166,42 +162,48 @@ const VulnerabilitiesGrid = ({ resources, onUpdate }) => {
                               value: label
                             })
                           }
+                          aria-label={arialabel}
                           aria-expanded={
                             textinputs
-                              ? selectedVulnerabilities
+                              ? grid.vulnerabilities[cbId]
+                                ? 'true'
+                                : 'false'
                               : undefined
```

Comment:
What's the difference between true and undefined? Can't it just be {texinputs && grid.vulnerabilities[cbId]} ?